src/master_element/MasterElementHO.C

```diff
@@ -260,7 +257,7 @@ void
 HigherOrderQuad3DSCS::area_vector(
   const double* POINTER_RESTRICT elemNodalCoords,
   const double* POINTER_RESTRICT shapeDeriv,
-  std::array<double,3>& areaVector) const
+  Kokkos::View<double[3]> areaVector) const
```

Comment:
Kokkos::Array<double,3>& .  Kokkos::View may heap allocate.

Reply:
Ah, yeah I'll move back to std::array in my next pr